class/defaults.yml

```diff
@@ -105,6 +105,8 @@ parameters:
     clusterManagementSystem:
       controlPlaneCluster: true
       serviceCluster: true
+      # This value should only be non-empty if control and service plane split is used
```

Comment:
In my opinion this comment should not exist, this gets things more complicated. We need to use variables from `config/vars.jsonnet` to figure out if the component is in split mode. If we add a comment to every new variable here, it's gonna become a mess. I am also not very happy to use those variable everywhere but an we find a solution?

Reply:
I've added asserts now, so it will scream at you if you're doing it wrong.